src/main/java/com/epam/test_generator/entities/IssueToDeleteInJira.java

```diff
@@ -0,0 +1,48 @@
+package com.epam.test_generator.entities;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import java.util.Objects;
+
+@Entity
+public class IssueToDeleteInJira {
```

Comment:
This class name is too verbose. Try to simplify it. For example `RemovedIssue`. Or `PendingRemovalIssue`.